src/utils/message.ts

```diff
@@ -0,0 +1,80 @@
+export const MessageType: {[type: string]: number} = {
```

Comment:
Maybe take a look at some [alternatives](https://maxheiber.medium.com/alternatives-to-typescript-enums-50e4c16600b1) to enums.
[Record Type](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype)

If you just remove the type it will be more type safe:
```typescript
export const MessageType = {
    UI_GET_DATA: 0,
};
```

Reply:
Done!